raiden_contracts/tests/test_token_network.py

```diff
@@ -18,6 +18,7 @@ def test_version(token_network):
     assert token_network.functions.contract_version().call() == CONTRACTS_VERSION
 
 
+@pytest.mark.skip(reason="Only for v0.12.0 release")
 def test_constructor_call(
```

Comment:
For those you skip like this I would say you should make a fixture that checks the releases and conditionally skips for the non-limits release numbers. Does not need to be part of this PR.

Reply:
That kind of sophisticated things will be tracked in https://github.com/raiden-network/raiden-contracts/issues/599#issuecomment-466200340 next week.

This PR is really for getting `raiden` developing going.